macOSUpgrade.sh

```diff
@@ -170,6 +198,41 @@ downloadInstaller() {
     /bin/kill "${jamfHUDPID}"
 }
 
+get_power_status() {
+    ##Check if device is on battery or ac power
+    ##If not, and our acPowerWaitTimer is above 1, allow user to connect to power for specified time period
+    if /usr/bin/pmset -g ps | grep "AC Power" > /dev/null ; then
+        /bin/echo "Power Check: OK - AC Power Detected"
+    else
+        if [[ -n "$acPowerWaitTimer" ]] && [[ "$acPowerWaitTimer" -gt 0 ]]; then
+            /Library/Application\ Support/JAMF/bin/jamfHelper.app/Contents/MacOS/jamfHelper -windowType utility -title "Waiting for AC Power Connection" -icon "$warnIcon" -description "Please connect your computer to power using an AC power adapter. This process will continue once AC power is detected." &
+            jamfHelperPowerPID=$!
+            wait_for_ac_power
+        else
+            sysRequirementErrors+=("• Is connected to AC power")
+            /bin/echo "Power Check: ERROR - No AC Power Detected"
+        fi
+    fi
+}
+
+get_free_space() {
```

Comment:
When I saw the process, I put errors into variables. Therefore, the naming 'get' seems to be inappropriate.
For example, how about naming such as 'validate'?

Reply:
Replaced get_ with validate_.
View [3c0067c](https://github.com/kc9wwh/macOSUpgrade/pull/115/commits/3c0067c1bbf88e2ab4059688ad4c12a183d5dc85)